x/auth/client/cli/query.go

```diff
@@ -327,6 +329,77 @@ $ %s query txs --%s 'message.sender=cosmos1...&message.action=withdraw_delegator
 	return cmd
 }
 
+// QueryBlocksByEventsCmd returns a command to search through blocks by events.
+func QueryBlocksByEventsCmd() *cobra.Command {
```

Comment:
Could you in the auth/README.md add docs about these new endpoints and commands?

Reply:
I'll add a `README.md` to the `auth/block` folder. I guess that's where it should go?

Reply:
Nope, in the auth readme itself there is a section for commands. This README is then rendered in docs.cosmos.network.

Reply:
I guess I was confused because the `auth/tx`  has a `Readme` with the `tx` cli commands that the `block` commands were modeled after.